include/zcbor_decode.h

```diff
@@ -440,6 +440,17 @@ void zcbor_bstr_next_fragment(zcbor_state_t *state,
 /** Can be used on any fragment to tell if it is the final fragment of the string. */
 bool zcbor_is_last_fragment(const struct zcbor_string_fragment *fragment);
 
+/** Diagnostic printing of CBOR data
```

Comment:
About the naming. I'm not against `diagnostic`, but do we support any other type of printing? If not, couldn't we simply call this `Printing of CBOR data` ? The act of printing a binary serialized data structure sort of implies that you are debugging/diagnosing already.

Reply:
There is already a `zcbor_print` function, but it's a logging function enabled by `ZCBOR_VERBOSE` so I could rename it to `zcbor_log`. zcbor_debug.h also contains a number of `zcbor_print_*()` functions. Renaming `zcbor_diag_print` to `zcbor_print_cbor` sounds like a good idea.

Reply:
I made these changes now. Most of them are in #373.